torch/cpu/__init__.py

```diff
@@ -55,8 +55,25 @@ class Stream:
     """
     N.B. This class only exists to facilitate device-agnostic code
     """
+    def __init__(self, priority: int = -1):
+        pass
 
-    pass
+    def wait_stream(self, stream) -> None:
+        pass
+
+
+class Event:
```

Comment:
Let's add a comment to this class since it's technically a public API:

```
    """
    N.B. This class only exists to facilitate device-agnostic code
    """
```
Maybe also need to add a doc entry for this